src/modules/Auth/components/Registration/Registration.css

```diff
@@ -0,0 +1,223 @@
+.primary-color{
+  background-color: #EF9B28 ;
+}
+
+.floating-form  {
+  display: flex;
+  flex-direction: column;
+  background-color: #314a59d4;
+  margin: auto;
+  max-width: 1400px;
+  padding: 15px 55px;
+  border-radius: 15px;
+}
+.logo-container{
+  text-align: center;
+}
+.img-fluid{
+  max-width: 100%;
+  height: auto;
+  width: 31%;
+}
+.auth-text-header{
+  line-height: 0.1;
+  color: #EF9B28 ;
+  position: relative;
+  font-weight: 700;
+}
+
+.auth-text-header::first-letter {
+  display: inline-block; 
+  border-bottom: 3px solid; 
+
+}
+
+
+  @media (min-width: 1200px) {
+    .form-inputs .username {
+      order: 1; /* Username input first */
+    }
+  
+    .form-inputs .email {
+      order: 2; /* Email input second */
+    }
+  
+    .form-inputs .phone {
+      order: 3; /* Phone number input third */
+    }
+  
+    .form-inputs .password {
+      order: 4; /* New password input fourth */
+    }
+  
+    .form-inputs .confirm-password {
+      order: 5; /* Confirm password input fifth */
+    }
+  
+    .form-inputs .country {
+      order: 6; /* Country input last */
+    }
+  }
+.profile-image {
+  position: relative;
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  margin-inline: auto;
+  background-color: #f1f1f1;
+  margin-block: 25px;
+}
+.btn {
+  width: 100%;
+}
+.btn:hover {
+  background-color: #ffb044;
+  border-color: #fe9300;
+}
+
+.profile-image .input-img {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  left: 4px;
+  top: 31%;
+  opacity: 0;
+  cursor: pointer;
+  z-index: 1;
+}
+
+
+.profile-image .profile-image-preview {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  border-radius: 50%;
+}
+
+.profile-image .icon-profile {
+  position: absolute;
+  top: 65px;
+  left: 65px;
+  width: 40%;
+  height: 40%;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #315951;
+  color: #EF9B28;
+  cursor: pointer;
+  z-index: 2;
+  font-size: 16px;
+}
+
+/****  floating-Lable style start ****/
+.floating-label {
+  position: relative;
+  margin-bottom: 40px;
+}
+.floating-input {
+  font-size: 14px;
```

Comment:
We should be using rem for font sizes for accessibility reasons as pixel size fonts don't increase when the user increase the default font-size of the browser.

Reply:
done